backendCore/dl/dl_eval.py

```diff
@@ -0,0 +1,39 @@
+import torch
+import torch.nn as nn
+
+
+def compute_correct(predicted, actual):

```

Comment:
might be good to have unit tests for these two functions (like use dummy tensors to ensure that this function works as intended)? could be a hopefully straightforward task for a dev to pick up